include/mbgl/ios/MapboxGL.h

```diff
@@ -2,3 +2,10 @@
 #import "MGLMapView.h"
 #import "MGLTypes.h"
 #import "MGLUserLocation.h"
+
+@interface MapboxGL : NSObject
```

Comment:
This header was meant to be an [umbrella header](https://developer.apple.com/library/mac/documentation/MacOSX/Conceptual/BPFrameworks/Tasks/IncludingFrameworks.html), which does nothing but import all the other public-facing headers. (A developer should only ever have to import the one umbrella header unless they’re doing something out of the ordinary.)

I’d move this class to another header and rename it `MGLAccountManager` or somesuch, sticking to the framework’s established prefix.